integration/integration_test.go

```diff
@@ -1596,6 +1599,111 @@ COPY --from=prebuild /the-future/hello.txt /the-future/hello.txt
 		require.Equal(t, "hello from the past\nhello from the future\n/the-past/hello.txt", strings.TrimSpace(out))
 	})
 
+	t.Run("MultistageRunCopyMinimal", func(t *testing.T) {
+		t.Parallel()
+
+		srv := gittest.CreateGitServer(t, gittest.Options{
+			Files: map[string]string{
+				"Dockerfile": fmt.Sprintf(`
+FROM %[1]s AS a
+RUN date > /date.txt
+FROM %[1]s
+COPY --from=a /date.txt /date.txt
```

Comment:
I didn't actually mean to include this test; I can remove it if folks feel strongly. It doesn't really expand on anything from the above "hello from the future" test.